### DISCUSSION TITLE
GFM-style callout syntax

### DISCUSSION THREAD

**Discussion by evicinelli** [quarto-dev/quarto-cli#9915] [Feature Requests]
### Description

Hi!
Heavy quarto user, not a programmer :)

Would it be possible to integrate the [syntax that gfm uses](https://github.com/orgs/community/discussions/16925#discussion-4085374) to produce callouts in quarto? Right now quarto uses a div approach to write callout blocks, which is wonderful; gfm callouts, though, are now correctly displayed in a variety of 3rd party renderers. In this way, a document written to produce quarto output would still be renderable in another viewer.

Should this be a quarto feature or a pandoc feature? Would it be cumbersome to add another syntax on top of the existing one for this feature?
Apologies if this has already been discussed, I've not found anything of the sort :)

**Comment by mcanouil** [comment]
It's already the case for output.
And it is very unlikely the Quarto syntax will be changed as it will be a breaking change that brings less. Same as adding another syntax with less capabilities.
Quarto callouts feature is more advanced that GFM callouts.

<img width="1624" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/8896044/5c4902bb-080b-4518-9df1-37fa6842568f">

**Reply by mcanouil** [reply]
Small additional question: what do you think the GFM syntax brings and why do you think having this syntax in addition to the div syntax can improve/ease Quarto usage?

**Reply by evicinelli** [reply]
Hi, sorry for late reply.
Gfm callouts are natively rendered in some 3rd party markdown viewer/editors: having a second options for callouts would allow the qmd file to be correctly displayed even in the editors that do not strictly support quarto syntax. I get that would be additional code to have a less powerful feature, though
